specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/virtualWan.json

```diff
@@ -3482,6 +3889,34 @@
         }
       },
       "description": "Vpn Profile Response for package generation."
+    },
+    "VpnLinkBgpSettings": {
+      "properties": {
+        "asn": {
```

Comment:
Will these acronyms be known to your users?

Reply:
Yes, ASN is a well known term for BGP.

Reply:
thanks for confirming